### ISSUE TITLE
Map to open on location

### ISSUE THREAD

**Issue by Willschubertclimbs** [theCrag/tca#139]
Would love the map to open where I am in the world

**Comment by bkucsera** [comment]
Can you please specify this demand?
i guess you mean something else than to have the "my location" button.

**Comment by rouletout** [comment]
Yes map should center on your current location and not somewhere in europe

**Comment by bkucsera** [comment]
Fixed.

**Comment by rouletout** [comment]
Hmm mine opened now in Africa with south on top of the map...

![image](https://user-images.githubusercontent.com/19162952/125625065-cac6b071-7850-4d93-8a91-4feec874210b.png)

**Comment by bkucsera** [comment]
Fixed, please check if it works for you.

**Comment by rouletout** [comment]
Does not work for me, now opens somewhere in Germany

**Comment by bkucsera** [comment]
We add a default GPS in Germany, if the device does not have permission for location data.
Are you sure you allow the app to use your location? Does it still above Germany after a second start?

**Comment by rouletout** [comment]
No, Malí again on second start

**Comment by Willschubertclimbs** [comment]
Hi there,

Mine still opens up in Europe while I am in Australia
Locations services are turned on

**Comment by bkucsera** [comment]
Hmm it is strange. Once again please check and retry the following steps:
- delete Topo Guru app form your iPhone
- reinstall from TestFlight. Please check if you install the latest version: 4.0.3 (9) <-- 9 is also important
- after the first start, set location permission to "Always while using the app"

After this you should see the map zoomed above your actual location.
Q: If you push "Show my location" button on the lower right corner of the screen, does it work?

Thanks for trying it again! (We've checked on 15 different iOS devices and everything works. Would be nice to find out the reason of this.)

**Comment by bkucsera** [comment]
> Hi there, Mine still opens up in Europe while I am in Australia Locations services are turned on
> […](#)
> Sent from my iPhone
> On 16 Jul 2021, at 2:10 am, rouletout ***@***.***> wrote: ﻿ No, Malí again on second start — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Hi!
Can i ask you to send your exact iphone device number and ios version number?
Do you still have the problem?
The "Show my location" button works for you? So you can see your actual location on the map?
Thanks for the help.

**Comment by rouletout** [comment]
@Willschubertclimbs please provide the details as requested by Balint, thanks

**Comment by Willschubertclimbs** [comment]
Hi there,

I have an iPhone 12pro

My location services are on and my map opens up at Munich and I am in Sydney Australia


Thanks
